source/contributing/how-to-migrate.md

```diff
@@ -0,0 +1,34 @@
+# How to Migrate Documentation - Step-by-Step Process
+
+Migrating documentation is often crucial when reorganizing any project. As such, below is a list of instructions and guidelines to aid you when embarking on the migration journey.
+
+1. **Licensing**

```

Comment:
```suggestion
## Licensing
```
These top level ordered list items should just be sections without numbers in the titles.